### ISSUE TITLE
Add a team page

### ISSUE THREAD

**Issue by tapaswenipathak** [OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io#100]
This will have a journey design, starting from the first person to the recent changes in the team with their responsibilities. I wish to take it up but after some time and I already have to work on enabling search on the website[high prio than this]. If someone is interested to work on this please contact me on Slack channel. 

Happy to have design suggestions for this page which goes with the theme of our main website.

**Comment by vaibhavsingh97** [comment]
@anubhakushwaha @tapasweni-pathak 
These are some design inspiration i had compiled which i think to have in team page.

#### Design 1: 
![image](https://user-images.githubusercontent.com/8705386/30259377-94ae0f6e-96b0-11e7-9930-438a2cf40e83.png)
![image](https://user-images.githubusercontent.com/8705386/30259395-a6bdd4aa-96b0-11e7-920c-456de4771347.png)
https://www.capshare.com/team

#### Design 2:
![image](https://user-images.githubusercontent.com/8705386/30259401-b586d9be-96b0-11e7-913f-bc67b250cf84.png)

#### Design 3:
![image](https://user-images.githubusercontent.com/8705386/30259533-5f6da908-96b1-11e7-9ace-08814bf18bbf.png)

#### Design 4:
![image](https://user-images.githubusercontent.com/8705386/30259952-e424d642-96b3-11e7-89f0-c530b5e3a0c5.png)
http://redcollar.digital/about/

#### Design 5:
![image](https://user-images.githubusercontent.com/8705386/30260095-d03967fa-96b4-11e7-93f2-f78fa3a952a6.png)
https://www.rezo-zero.com/en/agency/

#### Design 6:
![image](https://user-images.githubusercontent.com/8705386/30260114-fbfffc50-96b4-11e7-8287-9f3bc59c39ac.png)
https://www.fcinq.com/

#### Design 7:
![image](https://user-images.githubusercontent.com/8705386/30259923-ba79eaf8-96b3-11e7-80d9-bcd42bd7486d.png)

#### Design 8:
![image](https://user-images.githubusercontent.com/8705386/30260139-226cdb88-96b5-11e7-8a84-7bada9993e52.png)

#### Design 9:
![image](https://user-images.githubusercontent.com/8705386/30260248-bb6af3ec-96b5-11e7-8708-1011472de81f.png)

#### Design 10:
![image](https://user-images.githubusercontent.com/8705386/30260317-2c5901b6-96b6-11e7-85ac-d1ed28ea333a.png)
http://www.globulebleu.com/the-team/?lang=en

**Comment by vaibhavsingh97** [comment]
I am also thinking to have admin interface where `admin` can add new team members. what are your opinion on this?

**Comment by anubhakushwaha** [comment]
@vaibhavsingh97 Yes we can create a seperate issue for the creation of an admin interface for adding new team members, but then we will have to think of some dynamic UI where we can add our elements on go, what do you say?

**Comment by anubhakushwaha** [comment]
![oshc](https://user-images.githubusercontent.com/12814245/30422537-df617a82-995d-11e7-9ffa-665c25ab8de8.jpg)
I really liked the simplicity of this design, @tapasweni-pathak @vaibhavsingh97  @nikhita  @sandarumk  thoughts appreciated :)

**Comment by tapaswenipathak** [comment]
So many suggestions! 🌟 But we are not showing the journey of how people came in and are joining in. I wanted to have both history and team clubbed together. 

@OpenSourceHelpCommunity/everyone I have [this](https://www.mapbox.com/about/history/) in mind. Let me know! :)

**Comment by nikhita** [comment]
@anubhakushwaha Looks great! :smile: 

@tapasweni-pathak Nice idea! One question though: the Mapbox link shows how the company has progressed over the years (and people's contributions to that). We are still in our nascent stages so if we have a similar page...would it be a little too sparse? I like the idea but just want to be sure we don't end up with a negative connotations. :)

**Comment by tapaswenipathak** [comment]
We will show how we joined with how we started and helped with what tasks, less concentration on member, more on how and what they did with/to OSHC.

**Comment by sandarumk** [comment]
Does that mean a time line like this:
Started-> Created the slack group-> Started the Youtube Sessions-> Created the Website-> etc?

**Comment by tapaswenipathak** [comment]
brainstorm more. :) I will share in text form the plan, we can then work more on it.
